bigrig.js

```diff
@@ -94,6 +95,8 @@ try {
       strict: argv.strict
     });
   });
+
+  throw e;
```

Comment:
If you always throw e we always exit with an error, right? I wonder if a better way is to only swallow if the error occurs because of the `fs.stat` call. Other than that we probably want to throw it.